scripts/build-boshrelease-pipelines.sh

```diff
@@ -48,3 +48,5 @@ setup_release_pipeline rds-broker alphagov/paas-rds-broker-boshrelease master
 setup_release_pipeline datadog-for-cloudfoundry alphagov/paas-datadog-for-cloudfoundry-boshrelease master
 setup_release_pipeline logsearch-for-cloudfoundry alphagov/paas-logsearch-for-cloudfoundry gds_master
 setup_release_pipeline paas-haproxy alphagov/paas-haproxy-release master
+setup_release_pipeline graphite CloudCredo/graphite-statsd-boshrelease master
```

Comment:
This then opens the question if we should be keeping our fork, even if it is in sync with master, for the purpose of automatic builds...

I have also noticed that our final release versioning is completely arbitrary and doesn't match numbers of previous final releases. And the final release list is not updated after new one is build. Perhaps we should not call it 'final release' then, as this has a specific and significant meaning in BOSH, but e.g. 'master release'.